src/Range/Generic/Range`1.cs

```diff
@@ -18,5 +18,26 @@ public Range(IRange<T> value)
 
         public bool IsMinInclusive { get; set; }
         public bool IsMaxInclusive { get; set; }
+
+        // User-defined conversion from Digit to double 
+        public static implicit operator string(Range<T> range)
+        {
+            return range.ToString();
+        }
+        //  User-defined conversion from double to Digit 
+        public static implicit operator Range<T>(string range)
```

Comment:
Maybe this should be `explicit` since `Range.FromString<T>` will throw an exception if `range` is not properly formatted and convertible to `T`.

Ref. https://msdn.microsoft.com/en-us/library/xhbhezf4.aspx